app/src/main/java/com/github/se/gomeet/ui/mainscreens/events/MyEventInfo.kt

```diff
@@ -244,15 +260,25 @@ fun MyEventInfo(
                   Spacer(Modifier.height(screenHeight / 50))
                 } else {
                   Spacer(Modifier.height(10.dp))
-                  myEvent.value!!.posts.forEach { post ->
+                  posts.reversed().forEach { post ->
```

Comment:
I don't understand why each time posts is updated there is a call to reversed() when there is also a call to reversed() upon displaying the list of posts. Logically if reversed() is called upon initialisation/modification of posts, but also upon the usage of posts, wouldn't it make more sense to remove each reversed() call? (I mean that since the list gets reversed twice then it ends up as in the same order as if it weren't reversed at all, so what's the point of all the reverse calls?)

Reply:
I do need all of them, I tried to retrieve them cases by cases and I need them, otherwise when creating or deleting new posts, it won't keep them in reversed order. I do need the 4 reversed() call I made, you can try it by yourself if you want to convince yourself.